pombola/kenya/views_facebook_experiments.py

```diff
@@ -0,0 +1,200 @@
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.shortcuts import get_object_or_404
+from django.views.generic import TemplateView
+from django.conf import settings
+from django.core.cache import cache
+
+from mapit.models import Area, Generation
+
+from .election_data_2017.iebc_offices import IEBC_OFFICE_DATA
+from .forms import ConstituencyGroupedByCountySelectForm
+
+import requests
+import json
+
+
+AREA_ID_TO_OFFICE = {
+    o['cons_id']: o
+    for o in IEBC_OFFICE_DATA
+}
+
+
+class TreatmentOlympics(TemplateView):
+
+    template_name = 'facebook-experiment-olympics.html'
+
+    def get(self, request, *args, **kwargs):
+        self.survey_session = request.GET.get('sid')
+
+        if self.survey_session:
+            recordPositiveResponse(self.survey_session, '62', '10173')
```

Comment:
It would be good to define the magic integers here as helpfully named constants at the top of the module.